docs/bits/type/function.md

```diff
@@ -0,0 +1,25 @@
+**Function Domains**
+
+Takes 2 ```domains```(this page yet to be written) as arguments: the ```defined``` set and the ```range``` set of the function.
```

Comment:
instead of this put a (broken) link in. wikipedia does this...

or we could link to: https://conjure.readthedocs.io/en/latest/essence.html#domains

Reply:
done.